.drone.yml

```
@@ -0,0 +1,51 @@
+---
+
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+  - name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - ./gradlew assemble --no-daemon
+
+  - name: test project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - ./gradlew check --no-daemon
+    depends_on:
+      - build project
+
+trigger:
+  event:
+    - push
+
+---
+kind: pipeline
+type: kubernetes
+name: publish
+
+steps:
+  - name: test project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    commands:
+      - ./gradlew check --no-daemon
```

Do you need to `build project` prior to push? It might make more sense to run the `build` pipeline in both scenarios, and wait for it to finish before running `publish`:

```suggestion
trigger:
  event:
    - push
    - tag

---
kind: pipeline
type: kubernetes
name: publish
depends_on:
    - build

steps:
```